appcues/src/main/java/com/appcues/trait/appcues/SkippableTrait.kt

```diff
@@ -123,37 +160,171 @@ internal class SkippableTrait(
         content()
     }
 
-    /**
-     * Apply different modifiers based on ButtonAppearance
-     */
-    private fun Modifier.drawSkippableButton(buttonAppearance: ButtonAppearance): Modifier {
+    // styles the close X button differently based on the appearance set in config
+    private fun Modifier.styleButton(isDark: Boolean): Modifier {
         return this.then(
             when (buttonAppearance) {
-                Hidden -> Modifier
-                Default ->
+                ButtonAppearance.HIDDEN -> Modifier
```

Comment:
looking at this now, the HIDDEN case should be hoisted up even higher I think.  In `DecorateContainer`, it should not even add the `Spacer` at all, if HIDDEN, right?

Reply:
updated at top level to not include the `Spacer` when `hidden`

Reply:
I think before that I was basically not taking space, but this works better.